src/Commands/DefaultShooter.cpp

```diff
@@ -9,43 +9,55 @@
 // it from being updated in the future.
 
 
-#include "IntakeUp.h"
+#include "DefaultShooter.h"
 
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
 
-IntakeUp::IntakeUp(): Command() {
+DefaultShooter::DefaultShooter(): Command() {
         // Use requires() here to declare subsystem dependencies
     // eg. requires(chassis);
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
-	Requires(Robot::intake.get());
+	Requires(Robot::shooter.get());
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
 }
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
 
 // Called just before this Command runs the first time
-void IntakeUp::Initialize() {
+void DefaultShooter::Initialize() {
 
 }
 
 // Called repeatedly when this Command is scheduled to run
-void IntakeUp::Execute() {
-	RobotMap::intakeintakeMotor3->Set(0.5);
-	RobotMap::intakeintakeMotor4->Set(0.5);
+void DefaultShooter::Execute() {
+	RobotMap::shootershooterMotor1->Set(0.0);
+	RobotMap::shootershooterMotor2->Set(0.0);
+	RobotMap::shooterrollerMotor->Set(0.0);
+
+	if (RobotMap::shooterliftShooter->Get() == DoubleSolenoid::kForward)
```

Comment:
This class shouldn't handle the solenoids at all. Since it is created destroyed created, etc. If we set it to a default value once any command inturrupts it for example we want to put the shooter up it would go up then be set back down. A good place to put initial values for solenoids is Robot.cpp.